src/infi/clickhouse_orm/models.py

```diff
@@ -487,8 +487,8 @@ def create_table_sql(cls, db):
         '''
         Returns the SQL statement for creating a table for this model.
         '''
-        parts = ['CREATE TABLE IF NOT EXISTS `%s`.`%s` AS `%s`.`%s`' % (db.db_name, cls.table_name(), db.db_name,
-                                                                        cls.engine.main_model.table_name())]
+        parts = [f'CREATE TABLE IF NOT EXISTS `{db.db_name}`.`{cls.table_name()}` AS `{db.db_name}`.`{cls.engine.main_model.table_name()}` {db.on_cluster_sql} ('] 
```

Comment:
Why the ')' in the end?
```suggestion
        parts = [f'CREATE TABLE IF NOT EXISTS `{db.db_name}`.`{cls.table_name()}` AS `{db.db_name}`.`{cls.engine.main_model.table_name()}` {db.on_cluster_sql} '] 
```